Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -3702,6 +4054,18 @@ class LowerToAir {
             return;
         }
 
+        case Const128: {
+            auto value = m_value->as<Const128Value>()->value();
+            auto a = tmpForType(Int64);
+            auto result = tmp(m_value);
+            append(MoveZeroToVector, result);
+            append(Move, Arg::bigImm(value.u64x2[0]), a);
+            append(Air::VectorReplaceLaneInt64, Arg::imm(0), a, result);
+            append(Move, Arg::bigImm(value.u64x2[1]), a);
+            append(Air::VectorReplaceLaneInt64, Arg::imm(1), a, result);
+            return;
+        }
```

Comment:
I think better way of this handling is making it a load from memory, as the same to `double` constant handling.
See B3MoveConstants.cpp. And then, here, we only use `MoveZeroToVector`.